ztag/transforms/dns.py

```diff
@@ -1,10 +1,11 @@
 from ztag.transform import ZMapTransform, ZMapTransformOutput
 from ztag import protocols, errors
 
+CORRECT_RESPONSE = "192.150.186.1"
 
 class DNSTransform(ZMapTransform):
 
-    name = "dns/open_resolver"
+    name = "dns/open"
```

Comment:
This name is now wrong, and also deprecated. I think we can just delete this line?